Dockerfile-slim.template

```diff
@@ -3,8 +3,31 @@ FROM postgres:%%PG_MAJOR%%
 ENV POSTGIS_MAJOR %%POSTGIS_MAJOR%%
 ENV POSTGIS_VERSION %%POSTGIS_VERSION%%
 
```

Comment:
That's awesome, Tristan! I added one in d71f411, but I don't see any difference. Maybe it needs to be merged?

Also, see: https://github.com/github/linguist/blob/master/docs/overrides.md#using-gitattributes.

Reply:
The page I linked also says that I can use Vim or Emacs modelines to override the syntax on a particular file. Maybe that's an option here as well.

Reply:
It might take some time for the changes to be reflected as linguist rescans the project. I added YAML to the list of detected languages in one of my repositories and it took a few days to take effect 🤷‍♂️

Reply:
Looks like the changes have been reflected, I am seeing Dockerfile syntax highlighting in the file.